### ISSUE TITLE
XMB retroactive & retroactive_marked filename's incorrect

### ISSUE THREAD

**Issue by phantomsixthplayer** [libretro/retroarch-assets#49]
The following files are named incorrectly and because of that the images will display blank.
An update of the two following assets folders with name changes to the files will fix the blank image issue.

RetroArch\assets\xmb\retroactive\
RetroArch\assets\xmb\retroactive_marked\

NEC - PC Engine - TurboGrafx 16-content (E).png
NEC - PC Engine - TurboGrafx 16-content (J).png
NEC - PC Engine - TurboGrafx 16-content (U).png

Nintendo - Nintendo Entertainment System-content (E).png
Nintendo - Nintendo Entertainment System-content (J).png
Nintendo - Nintendo Entertainment System-content (U).png

Nintendo - Super Nintendo Entertainment System-content (E).png
Nintendo - Super Nintendo Entertainment System-content (J).png
Nintendo - Super Nintendo Entertainment System-content (U).png

Sega - Dreamcast-content (E).png
Sega - Dreamcast-content (J).png
Sega - Dreamcast-content (U).png

Sega - Mega Drive - Genesis-content (E).png
Sega - Mega Drive - Genesis-content (J).png
Sega - Mega Drive - Genesis-content (U).png

# Should be:

NEC - PC Engine - TurboGrafx 16 (E)-content.png
NEC - PC Engine - TurboGrafx 16 (J)-content.png
NEC - PC Engine - TurboGrafx 16 (U)-content.png

Nintendo - Nintendo Entertainment System (E)-content.png
Nintendo - Nintendo Entertainment System (J)-content.png
Nintendo - Nintendo Entertainment System (U)-content.png

Nintendo - Super Nintendo Entertainment System (E)-content.png
Nintendo - Super Nintendo Entertainment System (J)-content.png
Nintendo - Super Nintendo Entertainment System (U)-content.png

Sega - Dreamcast (E)-content.png
Sega - Dreamcast (J)-content.png
Sega - Dreamcast (U)-content.png

Sega - Mega Drive - Genesis (E)-content.png
Sega - Mega Drive - Genesis (J)-content.png
Sega - Mega Drive - Genesis (U)-content.png

**Comment by lasers** [comment]
@Kivutar Outdated issue?

**Comment by Kivutar** [comment]
I'm not sure really. We don't have any playlist that is named with those (E) (J) and (U) anyway. So those images will never be displayed.

**Comment by lasers** [comment]
I think the maker of `retroactive` and `retroarchive_marked` made everything to let users changing icons if they wanted to (according to their regions or such). You only see (U), (E), and (J) files in `./xmb/retroactive/` and `./xmb/retroactive_marked/` anyway.

If changing icons, one would omit/rename region names anyway.
```
Sega - Dreamcast (E).png
Sega - Dreamcast (J).png
Sega - Dreamcast (U).png
...to...
Sega - Dreamcast.png

Sega - Dreamcast-content (E).png
Sega - Dreamcast-content (J).png
Sega - Dreamcast-content (U).png
...to...
Sega - Dreamcast-content.png
```
I think this issue should be closed.

**Comment by Kivutar** [comment]
I agree.

**Comment by baxysquare** [comment]
I know this issue is closed but I missed seeing it.  I apologize.  As @Kivutar indicated, I have made extra icons for the theme that indicate region or color variations and they can be switched out manually as per @lasers instructions.  I tacked on the region code before the file extension but they can be renamed down the line if there ever comes a time when a playlist could distinguish via region code.  I don't think that feature is currently on the to-do list.
